pkgs/applications/networking/deck/default.nix

```diff
@@ -3,13 +3,13 @@ let
   short_hash = "86497a5";
```

Comment:
short_hash should be updated to `63b9742`.

Reply:
That seems like too much hassle to update manually. I actually would just use the version here...

Reply:
I agree it is a hassle. I just want to keep the nix output the same as the official one. 

Is it possible to set up something to check and change this value automatically?

Just one curl command will get it.

`curl -s https://api.github.com/repos/kong/deck/tags | jq -rc '.[] | select(.name=="v1.26.0")|.commit.sha' | head -c 7 -`

Reply:
Maybe this could be part of an "passthru.updateScript"?